integration_tests/mockApis/orchestration.ts

```diff
@@ -16,4 +17,25 @@ export default {
       },
     })
   },
+  stubPrisonerProfile: ({
+    prisonId,
+    prisonerId,
+    profile,
+  }: {
+    prisonId: string
+    prisonerId: string
+    profile: PrisonerProfilePage
```

Comment:
I think the type here should be `PrisonerProfile` from `orchestrationApiTypes` because that is what is returned from the API